src/examples/rest-with-auth/graphweaver-config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	adminUI: {
+		auth: {
+			primaryMethods: ['PASSWORD'],
+			secondaryMethods: ['API_KEY', 'ONE_TIME_PASSWORD', 'MAGIC_LINK', 'WEB3', 'PASSKEY'],
+			password: {
+				enableForgottenPassword: true,
+				enableResetPassword: true,
```

Comment:
Should these be default true? I feel like most people will want these and can turn it off if they like.